lambda_function.py

```diff
@@ -0,0 +1,71 @@
+import os
+import boto3
+import logging
+from botocore.exceptions import ClientError
+
+ses_client = boto3.client('ses')
+
+def lambda_handler(event, context):
+    file_urls = []
+
+    for record in event['Records']:
+        bucket = record['s3']['bucket']['name']
+        key = record['s3']['object']['key']
+
+        if "ons-dp-prod-encrypted-datasets" in bucket:
```

Comment:
Is it possible to just have the word "prod" here rather than the full bucket name?